src/GameConstantsAndTypes.cpp

```diff
@@ -30,6 +30,7 @@ static const char *RadarCategoryNames[] = {
 	"Air",
 	"Freeze",
 	"Chaos",
+	"Notes",
```

Comment:
So you were able to place the number of notes before `TapsAndHolds`? Didn't think you would.